### PR TITLE
022720

### PR DESCRIPTION
**Pull request by Tigerstripes2013**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/thevankeyboards/minivan/keymaps/jetpacktuxedo/config.h [hunk 1]
```diff
@@ -2,3 +2,12 @@
...
 #define RGBLIGHT_EFFECT_BREATHING
+
+#ifndef CONFIG_USER_H
+#define CONFIG_USER_H
+
+#include "../../config.h"
+
+#define TAPPING_TERM 500
+
+#endif
```
Comment (zvecr) [inline keyboards/thevankeyboards/minivan/keymaps/jetpacktuxedo/config.h]:
```suggestion
#define TAPPING_TERM 500
```

**Comment by Tigerstripes2013** [comment]
oh no, i am not jetpacktuxedo.  sorry, i am new to github and thought i did
that to my clone.  please do not merge.

Brandon

**Comment by zvecr** [comment]
I've closed this PR in that case, feel free to open another for a personal keymap if required.

**Comment by noroadsleft** [comment]
You can actually still fix this pretty easily.

Open your Terminal, navigate to the `qmk_firmware` directory, and type the following commands, pressing Enter after each:

```sh
cd keyboards/thevankeyboards/minivan/keymaps/
cp -r jetpacktuxedo/ tigerstripes2013/
git co 42d6270 -- jetpacktuxedo/
git commit -m "Restore jetpacktuxedo keymap for Minivan"
git add -- tigerstripes2013/
git commit -m "Add tigerstripes2013 keymap for Minivan"
git push
```

This copies the `jetpacktuxedo` keymap to `tigerstripes2013` with the modifications you've already made, restores the `jetpacktuxedo` keymap so it's unchanged, then adds the edited keymap under `tigerstripes2013`.

If you do this, just reply to this pull request and we can open it again (though we are closing the `master` branch to merges in a few hours, ahead of our next [Breaking Changes](https://docs.qmk.fm/#/breaking_changes) merge).

**Comment by Tigerstripes2013** [comment]
sorry, i've been at work.  i did as you said; thank you so much!  i'll pop
into the QMK discord eventually and create a new PR under my username.
again, thanks for your help and sorry for the mess!
Brandon

**Pull request closed without merging**